cmd/kubeadm/app/images/images.go

```diff
@@ -30,8 +30,13 @@ func GetCoreImage(image, repoPrefix, k8sVersion, overrideImage string) string {
 		return overrideImage
 	}
 	kubernetesImageTag := kubeadmutil.KubernetesVersionToImageTag(k8sVersion)
+	etcdImageTag := constants.DefaultEtcdVersion
+	etcdImageVersion, err := constants.EtcdSupportedVersion(k8sVersion)
```

Comment:
what happens on error here?
This probably requires a parameter change from `k8sVersion string` to `k8sVersion *version.Version` where `version` is `k8s.io/kubernetes/pkg/util/version`

Reply:
on line 33 etcdImageTag gets assigned default constants.DefaultEtcdVersion, if error occurs the default version is used, if not then etcdImageTag would get version from EtcdSupportedVersion.